rpc/documentation/api.md

```diff
@@ -792,6 +838,137 @@ wallet and republished later if it or a double spend are not mined.
 
 ___
 
+#### `TicketPrice`
+
+The `TicketPrice` method returns the price of a ticket for the next block, also 
+known as the stake difficulty. May be incorrect if the daemon is currently 
+syncing.
```

Comment:
Okay, after lengthy discussion I'll include a block height with the assumption that it could be inaccurate in the current API because it's polled in the daemon. In the future, when wallet is able to act asynchronously with daemon, we should be able to switch to a notification and update this to either the current state of the stake difficulty notifications or simply replace it with stake difficulty notifications. Along with dcrd/#285, that should alleviate ticket purchase UX woes.